.github/workflows/ci.yaml

```diff
@@ -14,7 +14,7 @@ on:
 
 jobs:
   check-for-new-cascadia:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest

```

Comment:
It's nice to run on latest but has a tendency to blow up builds unexpectedly when a new image becomes available. Can you fix this to your preferred verison

Reply:
Excellent suggestion :-) Thank you.